src/kibana/components/vislib/visualizations/line_chart.js

```diff
@@ -291,6 +296,7 @@ define(function (require) {
 
           width = elWidth - margin.left - margin.right;
           height = elHeight - margin.top - margin.bottom;
+          timeMarker = new TimeMarker(times, xScale, height);
```

Comment:
Why is the timeMarker being newed up here if there's a chance it's not going to be used? Why not new it up in the `if (addTimeMarker)` section?